build-system/eslint-rules/prefer-class-prop.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// Enforces 'class' over 'className' access in props.
```

Comment:
What about for tabIndex or the other props?

Reply:
That's a good point. `class` was most pressing because the others are less likely to be taken from the props. I'll add them in for the sake of completeness.